### PR TITLE
add vultr

### PR DESCRIPTION
**Pull request by Ketanest**
new folder
downloader.sh
changed readme

ad suggested by #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,13 @@ To allow GoogleBot, block all Google IP addresses and then allow the GoogleBot a
 - IPv6: https://raw.githubusercontent.com/lord-alfred/ipranges/main/cloudflare/ipv6.txt
 - IPv6 (merged): https://raw.githubusercontent.com/lord-alfred/ipranges/main/cloudflare/ipv6_merged.txt
 
+### CloudFlare
```
Comment (lord-alfred) [inline README.md:113]: `Vultr`

File: vultr/downloader.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+set -euo pipefail
+set -x
+
+
+# get from public ranges
+curl -s https://geofeed.constant.com/?json > /tmp/vultr.json
+
+
+# save ipv4
+jq '.prefixes[] | [.ip_prefix][] | select(. != null)' -r /tmp/vultr.json > /tmp/vultr-ipv4.txt
```
Comment (lord-alfred) [inline vultr/downloader.sh:15]: wrong path for ipv4 and ipv6:
```
$ curl -s https://geofeed.constant.com/?json > /tmp/vultr.json
$ jq '.prefixes[] | [.ip_prefix][] | select(. != null)' -r /tmp/vultr.json
jq: error (at /tmp/vultr.json:0): Cannot iterate over null (null)

$ jq '.ipv6_prefixes[] | [.ipv6_prefix][] | select(. != null)' -r /tmp/vultr.json
jq: error (at /tmp/vultr.json:0): Cannot iterate over null (null)
```

File: vultr/downloader.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+# https://docs.vultr.com/vultr-ip-space
+# https://geofeed.constant.com/?json
+
+set -euo pipefail
+set -x
+
+
+# get from public ranges
+curl -s https://geofeed.constant.com/?json > /tmp/vultr.json
```
Comment (lord-alfred) [inline vultr/downloader.sh:11]: Maybe download text: https://geofeed.constant.com/?text and split by `grep` (as another downloaders here)?

**Review by lord-alfred — CHANGES_REQUESTED** [review]

File: vultr/downloader.sh (hunk 2 above)
Comment (lord-alfred) [reply vultr/downloader.sh:15]: ipv4 and ipv6 path are the same, it's hard to separate them from json

File: vultr/downloader.sh (hunk 3 above)
Comment (Ketanest) [reply vultr/downloader.sh:11]: Will try

Comment (Ketanest) [reply vultr/downloader.sh]: should be fine now

File: vultr/downloader.sh [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+set -euo pipefail
+set -x
+
+
+# get from public ranges
+curl -s https://geofeed.constant.com/?text > /tmp/vultr.txt
+
+#save ipv4
+REGEX_V4="^(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)(\/([1-9]|[1-2][0-9]|3[0-2])){0,1}$"
```
Comment (lord-alfred) [inline vultr/downloader.sh:14]: too complicated and unreliable

example: https://github.com/lord-alfred/ipranges/blob/main/digitalocean/downloader.sh#L13-L17

**Review by lord-alfred — CHANGES_REQUESTED** [review]

File: vultr/downloader.sh (hunk 4 above)
Comment (Ketanest) [reply vultr/downloader.sh:14]: Just a "simple" IPv4 / v6 Filtering Regex. But okay, then I'll do like digitalocean.

**Review by lord-alfred — APPROVED** [review]

**Comment by lord-alfred** [comment]
Thank you, @Ketanest !

**Pull request merged**